tests/functional/Exasol/ExportTest.php

```diff
@@ -0,0 +1,225 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Keboola\Db\ImportExportFunctional\Exasol;
+
+use Keboola\CsvOptions\CsvOptions;
+use Keboola\Csv\CsvFile;
+use Keboola\Db\ImportExport\Backend\Exasol\ExasolImportOptions;
+use Keboola\Db\ImportExport\Backend\Exasol\Exporter;
+use Keboola\Db\ImportExport\Backend\Exasol\ToFinalTable\FullImporter;
+use Keboola\Db\ImportExport\Backend\Exasol\ToFinalTable\SqlBuilder;
+use Keboola\Db\ImportExport\Backend\Exasol\ToStage\StageTableDefinitionFactory;
+use Keboola\Db\ImportExport\Backend\Exasol\ToStage\ToStageImporter;
+use Keboola\Db\ImportExport\ExportOptions;
+use Keboola\Db\ImportExport\ImportOptions;
+use Keboola\Db\ImportExport\Storage;
+use Keboola\Db\ImportExport\Storage\S3;
+use Keboola\Db\ImportExport\Storage\ABS;
+use Keboola\TableBackendUtils\Table\Exasol\ExasolTableQueryBuilder;
+use Keboola\TableBackendUtils\Table\Exasol\ExasolTableReflection;
+
+class ExportTest extends ExasolBaseTestCase
+{
+    private const EXPORT_DIR = 'exasol_test_export';
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->clearDestination($this->getExportDir());
+        $this->cleanSchema($this->getDestinationSchemaName());
+        $this->cleanSchema($this->getSourceSchemaName());
+        $this->createSchema($this->getSourceSchemaName());
+        $this->createSchema($this->getDestinationSchemaName());
+    }
+
+    private function getExportDir(): string
+    {
+        $buildPrefix = '';
+        if (getenv('BUILD_PREFIX') !== false) {
+            $buildPrefix = getenv('BUILD_PREFIX');
+        }
+
+        return self::EXPORT_DIR
+            . '-'
+            . $buildPrefix
+            . '-'
+            . getenv('SUITE');
+    }
+
+    public function tearDown(): void
+    {
+        parent::tearDown();
+        unset($this->client);
+    }
+
+    public function testExportGzip(): void
+    {
+        // import
+        $schema = $this->getDestinationSchemaName();
```

Comment:
prijde mi, ze se ta hlavicka tady zbytecne opakuje, ale asi bych se na to vykaslal, nez delat nejake skopiciny jen kvuli tomu at znovupouzivame nejakych par radku...